src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-container.component.html

```diff
@@ -69,7 +69,7 @@
             ></jhi-programming-assessment-repo-export>
             <a
                 class="ml-2 mr-5"
-                href="{{ participation.repositoryUrl }}"
+                href="{{ adjustedRepositoryURL() }}"
```

Comment:
It's bad practice to invoke methods from the template. Due to change tracking those might be executed once every 10ms. Please only calculate this value once (e.g. as soon as the participation was loaded, save it in a variable `adjustedRepositoryUrl` in the component.ts and then directly use it here without a method

Reply:
done in #2376